osg_configure/configure_modules/infoservices.py

```diff
@@ -344,7 +344,7 @@ def _resourcecatalog_location(self):
 
 
 def ensure_valid_user_vo_file(using_gums, gums_host=None, logger=utilities.NullLogger):
-    if not validation.valid_user_vo_file(USER_VO_MAP_LOCATION):
+    if not (validation.valid_user_vo_file(USER_VO_MAP_LOCATION) and utilities.get_vos(USER_VO_MAP_LOCATION)):
```

Comment:
Not sure i understand what's being added here.  You are just using `utilities.get_vos` to attempt to parse the user vo map file?  Is `validation.valid_user_vo_file`'s validation not good enough in some cases?